frontend/src/styles.scss

```diff
@@ -1,6 +1,26 @@
+@use "../node_modules/@angular/material/" as mat;
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
 html, body { height: 100%; }
 body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; min-width: 21rem;}
+
+.optional-controls { display:flex; align-items: center; }
```

Comment:
where is this class used?

Reply:
This is used in the header-component.html to wrap with your language switcher feature in the mat-toolbar.

```
<div class="optional-controls">
    <app-language-switcher></app-language-switcher>
    <app-dark-mode-toggler></app-dark-mode-toggler>
  </div>
```